published/external/xdpapi.h

```diff
@@ -28,9 +34,12 @@ typedef enum _XDP_CREATE_PROGRAM_FLAGS {
 DEFINE_ENUM_FLAG_OPERATORS(XDP_CREATE_PROGRAM_FLAGS);
 C_ASSERT(sizeof(XDP_CREATE_PROGRAM_FLAGS) == sizeof(UINT32));
 
+#if defined(_KERNEL_MODE)
+
 typedef
-HRESULT
+XDP_STATUS
 XDP_CREATE_PROGRAM_FN(
+    _In_ VOID *ProviderBindingContext,
```

Comment:
Should an incomplete type be used here instead of a `void*`?

Reply:
Added XDP_API_PROVIDER_BINDING_CONTEXT